utils/keyprovider/keprovider.pb.go

```diff
@@ -0,0 +1,243 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// source: proto.proto
+
```

Comment:
could you add a `make` command to regenerate the protobuf? and in future we can add that to the CI checks

Reply:
done